lesson5/Emulator/Device.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Emulator
+{
+    public class Device : IDisposable, IDevice
```

Comment:
```suggestion
    public class Device : IDevice, IDisposable
```

IDisposable обычно последним указывают